src/screens/internal/CompareSnapshots/OptionsSelector.tsx

```diff
@@ -39,7 +39,7 @@ interface OptionsSelectorProps {
  * page (all graphs, etc.) to re-render causing a big perf hiccup.  See:
  * https://stackoverflow.com/q/60682426
  */
-const Select: React.FunctionComponent<SelectProps> = ({
+export const Select: React.FunctionComponent<SelectProps> = ({
```

Comment:
As a general guide, I usually try to avoid having one "screen" or "component" import code from another "screen" or "component."  If it's code that we want to share it's usually worth moving to its own component or into a shared source file (under common or whatever) depending on the code in question.

That said, since these are both internal tools that are kinda' related and the compare tool is already a bit messy, I wouldn't worry about it too much.  So I wouldn't bother changing this.  Just pointing it out for future reference.

Reply:
Ah, yeah that does make a lot of sense. Noted for the future and thanks for the flag :)